biztalk/install-and-config-guides/configure-biztalk-server.md

```diff
@@ -260,15 +262,16 @@ BAM alerts require BAM tools to be enabled.
 ### Configure BizTalk TMS
 
 >[NOTE]
->"Enterprise Single Sign-On Service"(SSO) is major dependency for TMS, TMS should be configured on server which has SSO service running. Also TMS is recommanded to configure only on one server in a BizTalk Group.
+>TMS should be configured on server which has SSO service running. Also TMS is recommanded to be configured only on one server in a BizTalk Group.
+>BizTalk TMS configuration is available start from **BizTalk Server 2020**.
 
 1. Select **BizTalk TMS**.
 2. Configure the following:
 
     |Use this|To do this|
     | --- | --- |
     |Enable BizTalk TMS | Check this if you want enable BizTalk TMS. | 
-    |Service accounts | Specify the account used to run TMS service.|
+    |Service accounts | Specify the account used to run TMS service. The service account should be a member of "SSO Administrator" group.|
```

Comment:
member of the "SSO Administrators" group
-> note 'the' and the name of the group has 's'